### PR TITLE
Small typo/grammar edits (Ready)

### LINKED ISSUE #22365: Shadow snare description has a typo
**Issue Description**:

image

**What did you expect to happen**: 

For it to be written "ensnares"

**What happened instead**:

It was "ensares"

**Why is this bad/What are the consequences**:

My eyes did burn because of grammar

**Steps to reproduce the problem**:

- Be a vampire and go into the "class" selection screen, notice that typo over shadow snare.

**When did the problem start happening**:

Since vampires were reworked

**Extra information**:

Good first issue if you wanna touch TGUI for the first time.

### PR DESCRIPTION
**Pull request by Oyu-De**
## What Does This PR Do
Fixes #22365 
Fixes typos and edits grammar in specmenu.js

## Why It's Good For The Game
Less typo

## Changelog
:cl:
spellcheck: Fixes typo and grammar on Vampire Spec menu
/:cl:

### CODE REVIEW

**Review by Henri215 — APPROVED** [review]

**Comment by Henri215** [comment]
use "spellcheck:" instead of "tweak:", and i would make it says something like.. "spellcheck: Fixes some grammar issues with vampire spec menu"

**Comment by Oyu-De** [comment]
> use "spellcheck:" instead of "tweak:", and i would make it says something like.. "spellcheck: Fixes some grammar issues with vampire spec menu"

Thx. Sorta rush made the pr at 5am sleep brain

File: tgui/packages/tgui/interfaces/SpecMenu.js [hunk 1]
```diff
@@ -222,8 +222,8 @@ const DantMenu = (props, context) => {
         <p>
           <b>Full Power</b>
           <Divider />
-          <b>Mass Hysteria</b>: Casts a powerful illusion that, blinds then make
-          everyone nearby perceive others to looks like random animals.
+          <b>Mass Hysteria</b>: Casts a powerful illusion that blinds then make
```
Comment (DGamerL) [inline tgui/packages/tgui/interfaces/SpecMenu.js:225]:
```suggestion
          <b>Mass Hysteria</b>: Casts a powerful illusion that blinds and then makes
```

**Review by DGamerL — APPROVED** [review]
Grammar team approves, just gotta rebuild TGUI :)

**Comment by Oyu-De** [comment]
> Grammar team approves, just gotta rebuild TGUI :)

I
Ah
AAAAAAAAAA TGUI WHY
I can do it i1-2 hours later, mind if I ping you on discord then?

**Review by Burzah — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**